builder/comp-builder.nix

```
@@ -408,6 +408,10 @@ let
         $SETUP_HS register --gen-pkg-config=${name}.conf
         ${ghc.targetPrefix}ghc-pkg -v0 init $out/package.conf.d
         ${ghc.targetPrefix}ghc-pkg -v0 --package-db ${configFiles}/${configFiles.packageCfgDir} -f $out/package.conf.d register ${name}.conf
+        if ! compgen -G "$out/package.conf.d/${name}-*.conf" > /dev/null; then
```

Actually, could we put this test in the `installCheckPhase`?

And can we add some logging to confirm that this line of script actually got executed?
```suggestion
        if compgen -G "$out/package.conf.d/${name}-*.conf" > /dev/null; then
            echo $out/package.conf.d/${name}-*.conf " is present"
        else
```